central/cluster/datastore/telemetry.go

```diff
@@ -37,11 +44,11 @@ func makeClusterProperties(cluster *storage.Cluster) map[string]any {
 
 func trackClusterInitialized(cluster *storage.Cluster) {
 	if cfg := centralclient.InstanceConfig(); cfg.Enabled() {
-		cfg.Telemeter().Group(cfg.GroupID, cluster.GetId(), nil)
-		cfg.Telemeter().Identify(cluster.GetId(), makeClusterProperties(cluster))
-		cfg.Telemeter().Track("Secured Cluster Initialized", cluster.GetId(), map[string]any{
-			"Health": cluster.GetHealthStatus().GetOverallHealthStatus().String(),
-		})
+		// Issue an event that makes the secured cluster identity effective:
+		cfg.Telemeter().TrackUserAs(cluster.GetId(), cluster.GetId(), "Secured Cluster",
```

Comment:
Probably want `Secured Cluster` as a `const` here, seeing that it's reused multiple times.

Reply:
makes sense!